diff --git a/src/project-graph/create-dependencies.js b/src/project-graph/create-dependencies.js
index b311ea87580ece8fe6928dc82e71c5ce9f7501d0..e7c9bdcb3060fa76abd9ffff2c2ad25993cf9c07 100644
--- a/src/project-graph/create-dependencies.js
+++ b/src/project-graph/create-dependencies.js
@@ -52,7 +52,7 @@ async function collectDependencies(node, context, project, filePath, collectedDe
         importLocator.fromNode(filePath, sourceFile, (importExpr, filePath, type) => {
             var _a;
             // locate project containing the import
-            const target = (_a = targetProjectLocator.findProjectWithImport(importExpr, filePath)) !== null && _a !== void 0 ? _a : `npm:${importExpr}`;
+            const target = (_a = targetProjectLocator.findProjectFromImport(importExpr, filePath)) !== null && _a !== void 0 ? _a : `npm:${importExpr}`;
             // add the explicit dependency when the target project was found
             collectedDependencies.push({
                 source: project,
